seatunnel-connectors-v2/connector-maxcompute/src/main/java/org/apache/seatunnel/connectors/seatunnel/maxcompute/util/MaxcomputeTypeMapper.java

```diff
@@ -0,0 +1,189 @@
+        String maxcomputeType = columnSchemaList.get(colIndex).getTypeInfo().getTypeName().toUpperCase();
+        switch (maxcomputeType) {
+            case MAXCOMPUTE_BOOLEAN:
+                return BasicType.BOOLEAN_TYPE;
+            case MAXCOMPUTE_TINYINT:
+            case MAXCOMPUTE_SMALLINT:
+            case MAXCOMPUTE_INT:
+                return BasicType.INT_TYPE;
+            case MAXCOMPUTE_BIGINT:
+                return BasicType.LONG_TYPE;
+            case MAXCOMPUTE_DECIMAL:
+                return new DecimalType(PRECISION, 0);
+            case MAXCOMPUTE_FLOAT:
+                return BasicType.FLOAT_TYPE;
+            case MAXCOMPUTE_DOUBLE:
+                return BasicType.DOUBLE_TYPE;
+            case MAXCOMPUTE_VARCHAR:
+            case MAXCOMPUTE_CHAR:
+            case MAXCOMPUTE_STRING:
+                return BasicType.STRING_TYPE;
+            case MAXCOMPUTE_DATE:
+            case MAXCOMPUTE_DATETIME:
+            case MAXCOMPUTE_TIMESTAMP:
+                return LocalTimeType.LOCAL_DATE_TIME_TYPE;
+            case MAXCOMPUTE_BINARY:
+                return PrimitiveByteArrayType.INSTANCE;
+            //Doesn't support yet
+            case MAXCOMPUTE_UNKNOWN:
+            default:
+                throw new MaxcomputeConnectorException(CommonErrorCode.UNSUPPORTED_DATA_TYPE, String.format(
+                    "Doesn't support Maxcompute type '%s' .",
+                    maxcomputeType));
+        }
+    }
+
+    public static SeaTunnelRowType getSeaTunnelRowType(Config pluginConfig) {
+        Table table = MaxcomputeUtil.getTable(pluginConfig);
+        TableSchema tableSchema = table.getSchema();
+        ArrayList<SeaTunnelDataType<?>> seaTunnelDataTypes = new ArrayList<>();
+        ArrayList<String> fieldNames = new ArrayList<>();
+        try {
+            for (int i = 0; i < tableSchema.getColumns().size(); i++) {
+                fieldNames.add(tableSchema.getColumns().get(i).getName());
+                seaTunnelDataTypes.add(mapping(tableSchema.getColumns(), i));
+            }
+        } catch (Exception e) {
+            throw new MaxcomputeConnectorException(CommonErrorCode.TABLE_SCHEMA_GET_FAILED, e);
+        }
+        return new SeaTunnelRowType(fieldNames.toArray(new String[fieldNames.size()]), seaTunnelDataTypes.toArray(new SeaTunnelDataType<?>[seaTunnelDataTypes.size()]));
+    }
+
+    public static SeaTunnelRow getSeaTunnelRowData(Record rs, SeaTunnelRowType typeInfo) throws SQLException {
+
+        List<Object> fields = new ArrayList<>();
+        SeaTunnelDataType<?>[] seaTunnelDataTypes = typeInfo.getFieldTypes();
+        for (int i = 0; i < seaTunnelDataTypes.length; i++) {
+            Object seatunnelField;
+            SeaTunnelDataType<?> seaTunnelDataType = seaTunnelDataTypes[i];
+            if (null == rs.get(i)) {
+                seatunnelField = null;
+            } else if (BasicType.BOOLEAN_TYPE.equals(seaTunnelDataType)) {
```

Comment:
Use switch(seatunnelDataType.getSqlType()) is better. You can refer to https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/reader/ParquetReadStrategy.java#L128